src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c

```diff
@@ -4,6 +4,8 @@
  * it is licensed under the BSD 3-Clause license.
  */
 
+#include <socket.h>
```

Comment:
It does fail when I try to compile it without <socket.h>:

```
>> WARNING: object file specified, but linking not enabled
[xi_bsp_time_cc3220_sntp.c]
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 194: warning #225-D: function "htonl" declared implicitly
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 222: error #71: incomplete type is not allowed
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 223: error #71: incomplete type is not allowed
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 224: error #20: identifier "socklen_t" is undefined
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 225: error #71: incomplete type is not allowed
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 230: error #71: incomplete type is not allowed
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 242: warning #225-D: function "socket" declared implicitly
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 242: error #20: identifier "AF_INET" is undefined
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 242: error #20: identifier "SOCK_DGRAM" is undefined
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 254: warning #225-D: function "htons" declared implicitly
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 254: error #20: identifier "INADDR_ANY" is undefined
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 255: warning #225-D: function "htonl" declared implicitly
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 257: warning #225-D: function "bind" declared implicitly
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 283: warning #225-D: function "setsockopt" declared implicitly
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 283: error #20: identifier "SOL_SOCKET" is undefined
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 283: error #20: identifier "SO_RCVTIMEO" is undefined
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 294: warning #225-D: function "sendto" declared implicitly
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 305: warning #225-D: function "recvfrom" declared implicitly
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 337: warning #225-D: function "ntohl" declared implicitly
"/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c", line 344: warning #225-D: function "close" declared implicitly
10 errors detected in the compilation of "/Users/palantir/Work/Xively/xively-client-c/src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c".
```

May have to do with the SDK update.